modules/test/playwright/tests/object-web/objectDefinitions.spec.ts

```diff
@@ -88,4 +88,150 @@ test.describe('Manage object definitions through Model Builder', () => {
 			objectDefinition.id
 		);
 	});
+
+	test('can delete an published object definition by model builder', async ({
+		apiHelpers,
+		modalAddObjectDefinitionPage,
+		modelBuilderPage,
+	}) => {
+		const objectDefinition1 =
+			await apiHelpers.objectAdmin.postRandomObjectDefinition({
+				objectFolderExternalReferenceCode: 'default',
+				status: {code: 0},
+			});
+
+		const objectDefinition2Label = 'ObjectDefinition' + getRandomInt();
+
+		await modelBuilderPage.goto({objectFolderName: 'Default'});
+
+		await modelBuilderPage.createNewObjectDefinitionButton.click();
+
+		const objectDefinition2 =
+			await modalAddObjectDefinitionPage.createObjectDefinition(
+				objectDefinition2Label
+			);
+
+		await modelBuilderPage.clickToggleSidebarsButton();
+
+		await modelBuilderPage.clickFitViewButton();
+
+		await modelBuilderPage.clickObjectDefinitionActionsButton(
+			objectDefinition1.name
```

Comment:
can you also pass here the label instead of the name? because the function will search for the label and not the name (i know that on the test the name and the label are the same, but we need to try to make the test as readable and closer to reality as possible)